spec/truffle/signal_exception_spec.rb

```diff
@@ -44,4 +44,25 @@
 
     $?.termsig.should == Signal.list["PROF"]
   end
+
+  it "does not self-signal for VTALRM" do
+    IO.popen(ruby_cmd("raise(SignalException, 'VTALRM')"), err: [:child, :out]) { |out|
+      out.read.should include("for SIGVTALRM as it is VM reserved")
+    }
+    $?.termsig.should be_nil
+  end
+
+  it "self-signals for USR1 when running natively" do
+    skip unless TruffleRuby.native?
```

Comment:
I'd prefer `guard -> { TruffleRuby.native? } do` like in other specs.

Reply:
Fixed.